packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -239,6 +245,22 @@ export function DevtoolsView(props: DevtoolsViewProps): React.ReactElement {
 		};
 	}, [messageRelay, setSupportedFeatures, telemetryOptInLogger]);
 
+	React.useEffect(() => {
+		const handleRequest = (event) => {
```

Comment:
Is `event` used in any part of the hook?

Reply:
Good catch, it isn't. The handler just send the response event. Will fix